metadata-ingestion/docs/sources/databricks/README.md

```diff
@@ -1,12 +1,12 @@
 DataHub supports integration with Databricks ecosystem using a multitude of connectors, depending on your exact setup. 
 
-## Databricks Hive
+## Databricks Hive (old)
 
 The simplest way to integrate is usually via the Hive connector. The [Hive starter recipe](http://datahubproject.io/docs/generated/ingestion/sources/hive#starter-recipe) has a section describing how to connect to your Databricks workspace.
```

Comment:
we shouldn't say it's old, if it's also the simplest way

let's also put the unity docs first